update_compilers/install_compilers.sh

```diff
@@ -413,6 +413,15 @@ if [ ! -d "libs/boost_1_64_0" ]; then
     popd
     rm -rf /tmp/boost
 fi
+
+# nvcc
```

Comment:
Thanks for this: I think we'll have to unpack the deb file manually though. This unconditionally installs the package on any computer running the script. The script is not run as root, and can't apt-get install. Plus, on non-amazon instances this forces users to install cuda and then (super-dangerously!) removes their `/var/lib/apt/lists` !

I'm sorry it wasn't obvious, and again there's no documentation on this so I understand the issues here. I know this seems ungrateful but I can't merge this as-is: if only as if I were to do so it would break my home computer!